config/deploy.rb

```diff
@@ -1,10 +1,11 @@
-lock '3.4.0'
+# config valid only for Capistrano 3.1
+lock '3.2.1'
```

Comment:
not intentional - you can revert, not sure why (perhaps I deployed and have cap 3.4 in my system and needed to do that)